src/parsers.js

```diff
@@ -0,0 +1,56 @@
+/** A parser utility that transforms messages targeting various versions of the Feersum schema */
+export default class FeersumParser {
+    constructor({ version }) {
+        this.version = version;
+    }
+
+    parse(data) {
+        return parserVersionMap[this.version](data);
+    }
+}
+
+const parserVersionMap = {
+    '0.9': parser09,
+    '0.10': parser10
+};
+
+/**
+ * Parser function for Feersum Schema v0.9
+ * @link http://dev.feersum.io/static/help/transports/feersum09.html#message-send-data-format
```

Comment:
for future reference maybe some detail around what needs to be parsed per version?